### PR TITLE
[FIF-150] Rename project to Project Buzz

### PR DESCRIPTION
**Pull request by jleiva-gap**
- The name FIF or Fix-it-Friday is updated to Buzz.
- The database is renamed to EdFi_Buzz.
- Projects are renamed with the EdFi.Buzz prefix.
- Documentation and scripts updated.

### CODE REVIEW

File: EdFi.Buzz.Api/.history/.env_20200713115240 [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+BUZZ_API_DB_PORT = 5432
+BUZZ_API_DB_USERNAME ='postgres'
+BUZZ_API_DB_PASSWORD = 'P@ssw0rd'
+BUZZ_API_DB_DATABASE = 'FixItFriday'
+BUZZ_API_DB_SCHEMA = 'fif'
+BUZZ_API_HTTP_PORT = 3000
+
+GOOGLE_DISCOVERY=https://accounts.google.com/.well-known/openid-configuration
+GOOGLE_CLIENT_ID=126791611214-6s679gcund8aas06qs94p9l4h9crs466.apps.googleusercontent.com
+GOOGLE_SECRET=anAftR8Aaynvt0vr5I747Q3c
```
Comment (stephenfuqua) [inline EdFi.Buzz.Api/.history/.env_20200713115240:11]: I hope this isn't a real Google Secret. If it is, then a new secret needs to be generated right away to replace this.

We should never have real credential in here, and any `.env` file should only be a `.env.example` file.

**Comment by DavidJGapCR** [comment]
Some of the Services have instances of repositories with names like FixItFridayRepository. We should probably rename those as well.

**Comment by stephenfuqua** [comment]
@jleiva-gap you'll need to complete remove commit 263f954ad8a5bd9acb3c3b763fbd0f21d863eac2. It maybe simplest to create a new branch and cherry pick the commits you want to keep into that, and then redo the `git mv` commands.

**Comment by DavidJGapCR** [comment]
> @jleiva-gap you'll need to complete remove commit [263f954](https://github.com/Ed-Fi-Alliance-OSS/Fix-It-Friday/commit/263f954ad8a5bd9acb3c3b763fbd0f21d863eac2). It maybe simplest to create a new branch and cherry pick the commits you want to keep into that, and then redo the `git mv` commands.

The reason is that we need to remove the google info, right? @stephenfuqua

**Comment by stephenfuqua** [comment]
> > @jleiva-gap you'll need to complete remove commit [263f954](https://github.com/Ed-Fi-Alliance-OSS/Fix-It-Friday/commit/263f954ad8a5bd9acb3c3b763fbd0f21d863eac2). It maybe simplest to create a new branch and cherry pick the commits you want to keep into that, and then redo the `git mv` commands.
> 
> The reason is that we need to remove the google info, right? @stephenfuqua

Both reasons: remove the google info, and simply deleting the `.history` files does not remove them from the repo - it just removes them from the head of the branch.

**Pull request closed without merging**